Open/pages/info.vue

```diff
@@ -279,7 +279,25 @@ enum StageStatus {
     },
     head () {
         return {
-            title: "Corsace Open",
+            title: this.$store.state["open"].title,
+            meta: [
+                {hid: "og:site_name", property: "og:site_name", content: `Corsace Open - ${this.$store.state["open"].tournament.year}`}, 
+                {hid: "og:title", property: "og:title", content: `Corsace Open - ${this.$store.state["open"].tournament.year}`}, 
+                {hid: "og:url", property: "og:url", content: "https://open.corsace.io"}, 
+                {hid: "og:type", property: "og:type", content: "website"},                 
+                {hid: "og:description", property: "og:description", content: "Corsace Open is one of the largest 4v4 tournaments in osu! Hailing great competition and entertaining matches."},
+                {hid: "og:locale",property: "og:locale", content: "en_US"},
+                {hid: "og:image",property: "og:image", content: "/_nuxt/_/Assets/img/site/open/banner.png"},
```

Comment:
shouldnt use /_nuxt/_/ should instead use relative dir instead or whatever

so that it works to assign it to the right image after nuxt's bundler bundles the image for prod